corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -633,6 +633,8 @@ def _get_asynchronous_payload(self):
             self.async_restore_task_id_cache.set_value(task.id)
         try:
             response = task.get(timeout=self._get_task_timeout(new_task))
+            RedisExponentialBackoff.clear_key(
```

Comment:
I think you want this to be in the `get_async_restore_payload` task, otherwise this is just going to get cleared every time someone hits the endpoint, somewhat defeating the purpose..

Reply:
Ah, you are right. But yes, I would move it into task where the other caches are getting cleared.